frontend/src/components/create-podcast/create-podcast.tsx

```diff
@@ -0,0 +1,69 @@
+import { useForm, FieldValues } from 'react-hook-form';
+import { PodcastCreatePayload } from 'common/types/types';
+import { joiResolver } from '@hookform/resolvers/joi';
+import { ButtonType, DataStatus } from 'common/enums/enums';
+import { podcast as podcastActions } from 'store/actions';
+import { useAppSelector, useDispatch } from 'hooks/hooks';
+import { Input, Button } from 'components/common/common';
+import styles from './styles.module.scss';
+import { PodcastCreatePayloadKey } from 'jabber-shared/common/enums/enums';
+import { createPodcastSchema } from 'validation-schemas/validation-schemas';
+import logoCut from 'assets/img/logo-cut.svg';
+
+const CreatePodcast: React.FC = () => {
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FieldValues>({
+    defaultValues: {
+      [PodcastCreatePayloadKey.NAME]: '',
+      [PodcastCreatePayloadKey.USER_ID]: 7,
```

Comment:
move it to constants
frontend/src/components/create-podcast/common/constants.ts